app/src/main/config/settings_diagnostics.plist

```diff
@@ -118,7 +118,7 @@
 	<string>false</string>
 	<!--  -->
 	<key>startup_request_device_owner</key>
-	<string>true</string>
+	<string>false</string>
```

Comment:
Why are all these parameters changed to false?

Reply:
Same reason as above.